desktop/ui/src/main/resources/datacleaner-home/conf.xml

```diff
@@ -67,10 +67,18 @@
 		</combined>
 	</storage-provider>
 
-	<classpath-scanner>
-		<package recursive="true">org.datacleaner</package>
-		<package recursive="true">com.hi</package>
-		<package recursive="true">com.neopost</package>
-	</classpath-scanner>
-
+    <descriptor-providers>
+        <classpath-scanner>
+            <package recursive="true">org.datacleaner</package>
+            <package recursive="true">com.hi</package>
+            <package recursive="true">com.neopost</package>
+        </classpath-scanner>
+        <remote-components>
```

Comment:
For now I would comment the <remote-components> section, until datacleaner.org service will be available.